CHANGELOG.md

```diff
@@ -0,0 +1,54 @@
+# Changelog
+All notable changes to this project will be documented in this file.
```

Comment:
I noticed last week a changelog was created for RVF as it was missing one. I made me realise we didn't have one for this project. 

It was quite difficult thinking of what to write for version 1.0.0... happy to take any suggestions.